amundsen_application/static/js/components/TableDetail/SourceLink/index.tsx

```diff
@@ -3,9 +3,14 @@
 
 import * as React from 'react';
 
+import {
+  getDescriptionSourceDisplayName,
+  getDescriptionSourceIconPath,
+} from 'config/config-utils';
 import { logClick } from 'ducks/utilMethods';
 import AvatarLabel from 'components/common/AvatarLabel';
 import { TableSource } from 'interfaces';
```

Comment:
if you do `import { TableSource, ResourceType } from 'interfaces'; `,  would that work?

Reply:
yeah it would work. I would change it. Thanks